packages/roosterjs-content-model/lib/editor/ContentModelEditor.ts

```diff
@@ -28,9 +32,17 @@ export default class ContentModelEditor extends Editor implements IContentModelE
             selectionRange: this.getSelectionRangeEx(),
             alwaysNormalizeTable: true,
             ...(option || {}),
+            shouldSkipProcessElement: this.createShouldHandleElement(),
         });
     }
 
+    private createShouldHandleElement(): Partial<ShouldSkipProcessElementMap> | undefined {
```

Comment:
at least for now we haven't see other requirements to check if we should skip element. So I think we can just do this check simply when process element in `knownElementProcessor` and directly call isDelimiter() there.

Reply:
It should be safe enough to just skip delimiter since we checked its class and its content. There is nothing visible different from user perspective with or without it.